docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -0,0 +1,75 @@
+================================================
+Troubleshooting and Frequently Asked Questions
+================================================
+
+..
+    Need to update with info about checking lambda errors in UI https://github.com/dockstore/dockstore/issues/3530
+
+.. contents::
+   :local:
+   :depth: 2
+
+Why should I migrate my existing workflows to use GitHub Apps and a .dockstore.yml?
+----------------------------------------------------------------------------------------
+Installing our Dockstore GitHub App onto your Github repository or organization will automatically sync your workflow on Dockstore whenever code is pushed to GitHub.
+This means less manual work for workflow developers, and less waiting for content to update.
+
+This requires the addition of a ``/.dockstore.yml`` file to your repository on Github.
+This file contains workflow information such as workflow path, test parameter paths, etc. that Dockstore will use to setup
+the corresponding workflow on Dockstore. It's important to note, that you will need a ``/.dockstore.yml`` file on each branch of your Github
+repository if you want to sync multiple branches (versions) of your workflow.
+
+You can read more about it at :doc:`/getting-started/github-apps/github-apps`.
+
+How does this change my development flow?
+-------------------------------------------
+If you have a ``/.dockstore.yml`` file in your master or develop branches on GitHub, any new branches you create from these as your template
+will have a  ``/.dockstore.yml``.
+As long as your workflow is already registered on Dockstore, your new branches will automatically sync corresponding new versions of the workflow on Dockstore.
+
+*Note:* If you want to edit version information, such as workflow path, you will have to update the ``/.dockstore.yml`` file directly on the corresponding GitHub branch.
+You can no longer do this directly on Dockstore.
+
+How do I check if the Dockstore Github App was installed?
```

Comment:
This section could also mention the Show GitHub app logs feature. Will reveal syntax errors and what not.

Reply:
There's a general TODO note to revisit this doc and some other places to add that stuff when its done.